storybook/helpers/index.js

```diff
@@ -0,0 +1,27 @@
+"use strict"
```

Comment:
There are some compiled js/declaration files that shouldn't be here.

Reply:
😓 I'll understand this yarn clean stuff one day

Reply:
This is our bad - they should be in .gitignore and be removed with `yarn clean` (which they're not). I've got an issue and PR to clean some of these up https://github.com/cultureamp/kaizen-design-system/issues/1729